### PR TITLE
V14 QA Added Content acceptance tests

### PR DESCRIPTION
**Pull request by nhudinh0309**

### CODE REVIEW

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/CultureAndHostnames.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+  });
+
+  test.afterEach(async ({umbracoApi}) => {
+    await umbracoApi.document.ensureNameNotExists(contentName);
+    await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+    await umbracoApi.language.ensureNameNotExists(languageName);
+  });
+
+  test('can add culture', async ({umbracoApi, umbracoUi}) => {
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/CultureAndHostnames.spec.ts:37]:
```suggestion
  test('can add a culture', async ({umbracoApi, umbracoUi}) => {
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/CultureAndHostnames.spec.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    await umbracoUi.content.clickCultureAndHostnamesButton();
+    await umbracoUi.content.selectCultureLanguageOption(languageName);
+    await umbracoUi.content.clickSaveModalButton();
+
+    // Assert
+    const domainsData = await umbracoApi.document.getDomains(contentId);
+    expect(domainsData.defaultIsoCode).toEqual(isoCode);
+  });
+
+  test('can add domains', async ({umbracoApi, umbracoUi}) => {
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/CultureAndHostnames.spec.ts:49]:
```suggestion
  test('can add a domain', async ({umbracoApi, umbracoUi}) => {
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/RedirectManagement.spec.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    await umbracoUi.redirectManagement.clickDisableURLTrackerButton();
+    await umbracoUi.redirectManagement.clickDisableButton();
+
+    // Assert
+    // Verfiy that if renaming a published page, there are no redirects have been made
+    // rename the published content 
+    await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.content.openContent(contentName);
+    await umbracoUi.content.enterContentName(updatedContentName);
+    await umbracoUi.content.clickSaveAndPublishButton();
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/RedirectManagement.spec.ts:46]: Maybe rename the content by using the API

**Review by andr317c — CHANGES_REQUESTED** [review]
Had some comments

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/RedirectManagement.spec.ts (hunk 3 above)
Comment (nhudinh0309) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/RedirectManagement.spec.ts:46]: If I rename the content by using the API, there is no redirects have been made. The redirect URL will be made when I rename it in UI.

**Review by andr317c — APPROVED** [review]
LGTM!

**Pull request merged**